### ISSUE TITLE
Lesson 3 Template Literals

### ISSUE THREAD

**Issue by SherylHohman** [udacity/reactnd-issues#225]
(Easy Fix)

The note about Template Literals in Lesson 3, Module 3: Functional Components, really belongs in Lesson 3, Module 2: Pass Data with Props, just before the 3rd video in that section.  

We first made use of Template Literals in the 3rd video of Module 2: file src/ListContacts.js, commit https://github.com/udacity/reactnd-contacts-complete/commit/43add2a640214483b00d9ca491990bb86104501e

Indeed I spent quite a bit of time puzzled as to why I could not get my Avatar to show up.  Nor did I understand the syntax used for the URL (StackOverflow helped here).  It was not obvious that backticks, rather than single-quotes were being used.  Only by copying your file from the commit and "diffing" our two files, did I notice the difference.  Without the knowledge/idea that that backticks would ever be present, we can only "see" single quotes, particularly with certain fonts.

Basically - Great Note.  Fantastic intro with useful Links! - But it's a few videos, and an entire module,  too late :-)

**Comment by hbkwong** [comment]
Hi @SherylHohman, thanks for checking in! Great catch on the use of back-ticks before they're acknowledged in 3.3. We're in the process of making updates to the React Nanodegree program, so I'll make sure this is on our radar for a future release. Hope you're having a wonderful holiday season :]
